### ISSUE TITLE
Madeline 1: Clone this GitHub Repo

### ISSUE THREAD

**Issue by paolaperaza** [paolaperaza/vfastronauts#1]
### What

Let's clone this GitHub repo (paolaperaza/vfastronauts). This will download any files within this repo onto your own computer.

### How

1. Assign yourself to this issue ➡️ 
2. Follow instructions here: https://help.github.com/en/articles/cloning-a-repository
3. Close this issue once you're done!

### Pro-Tips

- Make sure you're in the right "directory" (aka folder) when you clone it
- Try creating an "AstronomerCode" folder on your Desktop (name it whatever you want - bonus points if you find out how to create a folder from your terminal), and then throw this GitHub repo in there
- To check where you are at any time, run `ls` in your terminal
- To jump into any folder you have access to from there (hierarchically lower), run `cd <folder name>`. E.g. `cd Desktop`
- If you're in a layer too deep, go one folder back up by running: `cd ..`
- When you run `ls` inside of `vfastronauts`, you should see:
   - `README.md`
   - `sample-doc`.md

```
Paolas-MacBook-Pro:~ paola$ cd Desktop
Paolas-MacBook-Pro:Desktop paola$ ls
vfastronauts
Paolas-MacBook-Pro:Desktop paola$ cd vfastronauts
Paolas-MacBook-Pro:vfastronauts paola$ ls
README.md
sample-doc
Paolas-MacBook-Pro:vfastronauts paola$ cd ..
Paolas-MacBook-Pro:Desktop paola$ 
```

**Need Help?** Tag me (@paolaperaza ) and ask away - I'll get a notification for it.

**Comment by madeline613** [comment]
@paolaperaza Since I'm using my own github account, I think I might need to be added as a collaborator in order to assign myself to issues?

**Comment by paolaperaza** [comment]
@madeline613 Yes! Ran into this with @Jake1021 but didn't know your username/the email you ended up going with.

Added you as a collab - lemme know if you have any issues.
